src/renderer/components/ft-channel-bubble/ft-channel-bubble.vue

```diff
@@ -1,11 +1,15 @@
 <template>
   <div
     class="bubblePadding"
+    tabindex="0"
+    :aria-labelledby="'bubble-' + channelName"
     @click="handleClick"
+    @keydown="handleClick($event)"
   >
     <img
       class="bubble"
       :src="channelThumbnail"
+      alt=""
```

Comment:
@jasonhenriquez which UI part is it? I can test if it's presented correctly.

Reply:
Thank you so much for helping with this @RastislavKish! This control is a link to a given user's channel. One place it appears many times is on the side navigation bar as a list of channels that someone is subscribed to. It may also appear under the "About" section of a given channel as one of its related channels.

Reply:
@jasonhenriquez so it should be read as the linked channel's name, right?
I can confirm this works correctly, I see all my subscribed channels listed on the sidebar, activating any of them brings me to the appropriate channel.

Feel free to ask if anything requires testing, I'm keeping my eye on this thread as well as the new commits, so I'm ready to try anytime!
Thanks for all the effort!

Reply:
Good, it's working as intended! You have been a spectacular help @RastislavKish. The app will be more accessible to so many more people because of your testing and input. You do not have to do any further work on this if you don't want to. The only other new changes at this time to test are modal accessibility (e.g., when using "Import Subscriptions" on the setting page) and left & right arrow key navigation of videos.